web/account_controller.go

```diff
@@ -0,0 +1,79 @@
+package web
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/asdine/storm"
+	"github.com/asdine/storm/q"
+	"github.com/gin-gonic/contrib/sessions"
+	"github.com/gin-gonic/gin"
+	"github.com/manyminds/api2go/jsonapi"
+	"github.com/smartcontractkit/chainlink/services"
+	"github.com/smartcontractkit/chainlink/store/models"
+	"github.com/smartcontractkit/chainlink/store/presenters"
+	"github.com/smartcontractkit/chainlink/utils"
+)
+
+// AccountController manages the current Session's User Account.
+type AccountController struct {
```

Comment:
Any reason this isn't UserController? Seems like a disconnect with the model/presenter.

Reply:
Yeah it could be UserController, I used Account to match with the existing usage of `account`.